haiku-apps/drive_encryption/drive_encryption-1.0.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A tool to encrypt devices and file images"
+DESCRIPTION="DriveEncryption is a suite of programs to allow you to \
+encrypt devices and file images. The encryption is done via the AES algorithm, \
+and protected via a password.
+It can also use devices created with TrueCrypt (http://www.truecrypt.org/), \
+available for Windows and Linux, as long as you've created them to use the \
+AES encryption algorithm and a RIPEMD-160 hash algorithm protected password."
+HOMEPAGE="https://github.com/chirayudesai/driveencryption"
+COPYRIGHT="2007-2014 pinc Software"
+LICENSE="
+	TrueCrypt
+	MIT
+	"
+REVISION="1"
+CHECKSUM_SHA256=""
+SOURCE_URI="https://github.com/axeld/driveencryption/archive/993a11ceefac15d77406b697afb60b177d99bc45.tar.gz"
+SOURCE_DIR="driveencryption-993a11ceefac15d77406b697afb60b177d99bc45"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	drive_encryption = $portVersion
+	cmd:encrypted_drive_control = $portVersion
+	app:DriveEncryption = $portVersion
+	app:Login = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:make
+	cmd:gcc
+	cmd:mkdepend
+	"
+
+BUILD()
+{
+	for i in lib driver bin gui login; do
+		pushd $i
+		make $jobArgs OBJ_DIR=objects
+		popd
+	done
+}
+
+INSTALL()
+{
+	mkdir -p $addOnsDir/kernel/drivers/bin $addOnsDir/kernel/drivers/dev/disk
+        cp -a driver/objects/encrypted_drive $addOnsDir/kernel/drivers/bin
```

Comment:
Sorry I missed that, turns out there was way too many spaces (not even using tabs). It should be fixed now.